app/src/androidTest/java/com/sdp/movemeet/view/navigation/NavigationTest.java

```diff
@@ -125,10 +147,74 @@ public void NavigationTest() {
         try{
             Thread.sleep(500);
         }catch(Exception e){}
+        intended(hasComponent(ActivityDescriptionActivity.class.getName()));
+
+        // Open Drawer to click on navigation.
+        onView(withId(R.id.drawer_layout)).check(matches(isClosed(Gravity.LEFT))).perform(DrawerActions.open());
+
+        try{
+            Thread.sleep(500);
+        }catch(Exception e){}
+
+        onView(withId(R.id.nav_list_activities)).perform(click());
+
+        try{
+            Thread.sleep(500);
+        }catch(Exception e){}
+        intended(hasComponent(ActivityListActivity.class.getName()));
+
+        // Open Drawer to click on navigation.
+        onView(withId(R.id.drawer_layout)).check(matches(isClosed(Gravity.LEFT))).perform(DrawerActions.open());
+
+        try{
+            Thread.sleep(500);
+        }catch(Exception e){}
+
+        onView(withId(R.id.nav_view)).perform(swipeUp());
+        onView(withId(R.id.nav_chat)).perform(click());
```

Comment:
Did you observe the apparition of the "`java.lang.RuntimeException: Uncaught exception in Firebase Database runloop (19.7.0). If you are not already on the latest version of the Firebase SDKs, try updating your dependencies`" error with this line of code? Do you think navigating to the chat here could induce this error?

Reply:
It is true that it seems that this error comes from the navigation to the cat. But, maybe by chance, I have never had a problem with this test yet. Fingers crossed that it lasts.

Reply:
Ok ok, effectively 🤞